cinema/serializers.py

```diff
@@ -1,16 +1,47 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Actor, Genre, CinemaHall, Movie
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Actor
+        fields = ["first_name", "last_name"]
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = ["name"]
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ["name", "rows", "seats_in_row"]
 
 
 class MovieSerializer(serializers.Serializer):
     id = serializers.IntegerField(read_only=True)
     title = serializers.CharField(max_length=255)
     description = serializers.CharField()
```

Comment:
You don't need to write all these fields in MovieSerializer
just inherit your serializer from ModelSerializer, like you did before. It should work
The only thing you have to keep is the ManyToMany fields, that's all
No other fields, no methods overriding